cmd/buildah/login.go

```diff
@@ -70,7 +70,7 @@ func loginCmd(c *cobra.Command, args []string, iopts *loginReply) error {
 	if err != nil {
 		return errors.Wrapf(err, "error reading auth file")
 	}
-	if !iopts.getLogin {
+	if iopts.getLogin {
```

Comment:
I think this line might be better as:  
```
if c.Flag("get-login").Changed {
```
and then leave line 52 set as true still.  

With only that change, I'm seeing:
```
# buildah login --get-login docker.io
tomsweeneyredhat
```
@QiWang19 WDYT?

Reply:
I agree. using `if c.Flag("get-login").Changed` and leave the default to true is compatible with podman.